syntaxhighlighter3/styles/shCore.css

```diff
@@ -95,6 +95,11 @@
 .syntaxhighlighter table td.code .container {
   position: relative !important;
 }
+.syntaxhighlighter table td.code .container:after,
```

Comment:
Cool! I think we could just add a comment here to document that it's just fixing a conflict with bootstrap CSS that can be used in some sites. WDYT?